lib/siwapp/schema/customer.ex

```diff
@@ -3,6 +3,18 @@ defmodule Siwapp.Schema.Customer do
   alias Siwapp.Schema.Invoice
   import Ecto.Changeset
 
+  @derive {Jason.Encoder, only:
+    [:name,
+    :identification,
+    :email,
+    :contact_person,
+    :active,
+    :deleted_at,
```

Comment:
Remove this field. This field is set when the customer is deleted and therefore ignored.